docs/copied-from-beats/shared-directory-layout.asciidoc

```diff
@@ -24,15 +24,16 @@ The directory layout of an installation is as follows:
 | logs   | The location for the logs created by {beatname_uc}. | {path.home}/logs | path.logs
```

Comment:
[Link](https://www.elastic.co/guide/en/apm/server/6.5/directory-layout.html). No changes if attributes stay the same.